dapp/src/components/TransactionModal/ActiveStakingStep/StakeFormModal/index.tsx

```diff
@@ -33,8 +34,20 @@ function StakeFormModal({
       {featureFlags.ACRE_POINTS_ENABLED && (
         <AcrePointsRewardEstimation mt={5} />
       )}
+
       <StakeDetails currency="bitcoin" />
-      <FormSubmitButton mt={10}>Deposit</FormSubmitButton>
+
+      <Box
+        pos="sticky"
+        bottom={-10}
+        m={-10}
+        mt={10}
+        p={10}
+        pt={0}
```

Comment:
Pagination? What kind of pagination do you mean?
I just built it over existing solution to prevent larger refactors.
Of course we can move the button if you wish.

Reply:
Oh, I'm sorry, 😆 I mean the padding for the modal window which is set in `modalTheme`.  Changing the padding in `modalTheme` also means changing the padding/margin in the container you have created. Check out the above screenshot. I think we can avoid it, I have something like this in my mind:

`ActionFormModal.tsx`

```
      <ModalBody px={10} boxSize="full">
        <FormComponent onSubmitForm={handleSubmitFormWrapper} />
      </ModalBody>
```

`TokenAmountFormBase.tsx`

```
    <Form
      id={formId}
      onSubmit={formikProps.handleSubmit}
      boxSize="full"
      display="flex"
      flexDirection="column"
      overflowY="hidden"
    >
      <Box overflowY="auto">
        <FormTokenBalanceInput
          name={TOKEN_AMOUNT_FIELD_NAME}
          tokenBalance={tokenBalance}
          placeholder={tokenBalanceInputPlaceholder}
          tokenAmountLabel={tokenAmountLabel}
          currency={currency}
          withMaxButton={withMaxButton}
          defaultValue={defaultAmount}
          autoFocus
        />
        {children}
      </Box>

      <FormSubmitButton mt={10}>{submitButtonLabel}</FormSubmitButton>
    </Form>
```

The result below:

https://github.com/user-attachments/assets/4783a01d-0936-4185-b509-cc7ca24fcdf4

Reply:
The fix will be done in another PR.